framework/include/variables/MooseLinearVariableFV.h

```diff
@@ -83,10 +83,15 @@ class MooseLinearVariableFV : public MooseVariableField<OutputType>
 
   virtual bool isFV() const override { return true; }
 
+  /**
+   * If the variable has a dirichlet boundary condition at face described by \p fi .
+   */
+  virtual bool isDirichletBoundaryFace(const FaceInfo & fi) const;
+
   /**
    * Switch to request cell gradient computations.
    */
-  void computeCellGradients() { _needs_cell_gradients = true; }
+  void computeCellGradients() const { _needs_cell_gradients = true; }
```

Comment:
I don't like this change and I don't think it's needed. AFAICT the only place you calls this is with a non-const reference to the variable

Reply:
In the `LinearFVMomentumPressure` I used a constant reference to the pressure. I'll change it to nonconst reference.